api/services/security/v1/schemas/SpotResponse_PaginatedResponse_AssetSimFilterModel__.yaml

```diff
@@ -0,0 +1,17 @@
+properties:
+  error:
+    anyOf:
+    - $ref: ./ErrorModel.yaml
+    - type: 'null'
+  request:
+    $ref: ./RequestModel.yaml
+  response:
+    allOf:
+    - $ref: ./PaginatedResponse_AssetSimFilterModel_.yaml
+  responseTime:
```

Comment:
is this API-approved? I am not familiar with returning this field in any other of our API routes